.github/workflows/UpdateALGoOnOfficialBranches.yaml

```diff
@@ -0,0 +1,51 @@
+name: Update AL-Go On Official Branches
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "55 08 * * 3" # Run at 08:55 UTC on Wednesday
+
+defaults:
+  run:
+    shell: powershell
+
+jobs:
+  GetBranches:
+    name: Get Official Branches
+    runs-on: windows-latest
```

Comment:
Could we run it on ubuntu-latest? Would be a bit fast I suppose

Reply:
Done.